src/main/java/sg/ncl/MainController.java

```diff
@@ -2283,11 +2301,24 @@ private void setSessionVariables(HttpSession session, String loginEmail, String
         session.setAttribute("sessionLoggedEmail", loginEmail);
         session.setAttribute("id", id);
         session.setAttribute("name", user.getFirstName());
+        // FIXME: get user type from token
+//        session.setAttribute("userType", "normal");
+        session.setAttribute("userType", "admin");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "userType" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)